Adding documentation for LLM feature (#239)
mozilla-l10n/localizer-documentation: src/tools/pontoon/ui.md

Be careful when using the machinery tab as suggestions may not match the source string for the project being translated. Even if the source strings match, the context in which strings are used may not be the same. This could lead to incorrect or unnatural translations. Always keep the meaning and purpose of the string being translated in mind when using the machinery tab.

#### Large language model (LLM) integration

> For all locales that use Google Translate as a translation source, a dropdown labeled `AI` becomes available. This feature refines the Google Translate output using a LLM. Collapsing this dropdown will reveal three options:

Reviewer: `... using an LLM.`

> For all locales that use Google Translate as a translation source, a dropdown labeled `AI` becomes available. This feature refines the Google Translate output using an LLM. Collapsing this dropdown will reveal three options:

Reviewer 2: ```suggestion
Pontoon will show a dropdown labeled `AI` for all locales that have Google Translate available as a translation source. This feature refines the Google Translate output using an LLM. Opening this dropdown will reveal three options:
```

* `REPHRASE`: Generate an alternative to this translation.
* `MAKE FORMAL`: Generate a more formal version of this translation.
* `MAKE INFORMAL`: Generate a more informal version of this translation.

> After selecting an option, the revised translation will replace the original suggestion. Once a new translation is generated, another option `SHOW ORIGINAL` will be seen in the dropdown menu. You can revert to the original translation by selecting the `SHOW ORIGINAL` option from the dropdown menu.

Reviewer 2: ```suggestion
After selecting an option, the revised translation will replace the original suggestion. Once a new translation is generated, another option `SHOW ORIGINAL` will be available in the dropdown menu. Selecting it will revert to the original suggestion.
```

![Dropdown to use the LLM feature](../../assets/images/pontoon/ui/llm_dropdown.png "Screenshot of the dropdown to use the LLM feature")

> The screenshot above displays the dropdown to use the LLM feature.

Reviewer 2: I would drop these descriptions of the screenshots (they're already in the alt text).

You can put the first screenshot after the list, before the SHOW original paragraph.
